include/rdma/fabric.h

```diff
@@ -269,7 +269,8 @@ enum {
 	FI_PROTO_GNI,
 	FI_PROTO_RXM,
 	FI_PROTO_RXD,
-	FI_PROTO_MLX
+	FI_PROTO_MLX,
+	FI_PROTO_NETWORKDIRECT
```

Comment:
Is there any way to determine the underlying protocol being used?  E..g. RoCE vs iWarp vs IB?